packages/menu/src/useMenu.ts

```diff
@@ -729,12 +749,39 @@ export const useMenu = <T extends HTMLElement = HTMLElement, M extends HTMLEleme
         'aria-checked': selected,
         'aria-disabled': itemDisabled,
         role: itemRole === null ? undefined : itemRole,
+        href,
         onClick,
         onKeyDown,
         onMouseEnter,
         ...other
       };
 
+      /**
+       * Validation
+       */
+
+      if (href && (isNext || isPrevious || type)) {
+        let invariantKey: string;
+
+        if (isNext) invariantKey = 'isNext';
+        else if (isPrevious) invariantKey = 'isPrevious';
+        else invariantKey = type!;
```

Comment:
Please structure with braces (I guess the eslint format rule that used to catch this for us is deprecated).